.eslintrc.json

```diff
@@ -8,7 +8,11 @@
         "project": ["tsconfig.json", "e2e/tsconfig.json"],
         "createDefaultProgram": true
       },
-      "extends": ["plugin:@angular-eslint/recommended", "plugin:@angular-eslint/template/process-inline-templates"],
+      "extends": [
+        "plugin:@angular-eslint/recommended",
+        "plugin:@angular-eslint/template/process-inline-templates",
+        "plugin:@typescript-eslint/recommended"
```

Comment:
This had to be included otherwise ESLint was complaining because it couldn't find some rules.
The plugin automatically imports these e.g.